documentation/api/query/v4/reports.markdown

```diff
@@ -326,6 +326,16 @@ This will return all events for a particular report, designated by its unique ha
 
 This is a shortcut to the [`/events`][events] endpoint. It behaves the same as a call to [`/events`][events] with a query string of `["=", "report", "<HASH>"]`.
 
+## `GET /v4/reports/<HASH>/metrics`
+
+This will return all metrics for a particular report, designated by its unique hash.
+This `metrics` endpoint does not currently support querying or paging.
```

Comment:
Are you saying "This" here because there may be another one eventually? Seems like "The" is more appropriate.